handler/dimensions.go

```diff
@@ -30,6 +31,81 @@ func NewDimensions(cfg *config.Config, r responder, c cantabularClient, d datase
 	}
 }
 
+// GetDimensionCategories is the handler for GET/population-types/{population-type}/dimension-categories?dims=xxx
+func (h *Dimensions) GetDimensionCategories(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	req := contract.GetDimensionCategoriesRequest{
+		PopulationType: chi.URLParam(r, "population-type"),
+		Variables:      r.URL.Query().Get("dims"),
+	}
+
+	logData := log.Data{
+		"request": req,
+	}
+
+	if err := parseRequest(r, &req); err != nil {
+		h.respond.Error(ctx, w, http.StatusBadRequest, &Error{
+			err: errors.Wrap(err, "failed to get dimension categories"),
```

Comment:
If you're not using the extra fields the error struct you can just return the error (errors.Wrap...)